starport/templates/typed/genesis.go

```diff
@@ -146,6 +148,54 @@ genesis.%[2]vCount = k.Get%[2]vCount(ctx)
 	}
 }
 
+func genesisTestsModify(replacer placeholder.Replacer, opts *Options) genny.RunFn {
+	return func(r *genny.Runner) error {
+		path := fmt.Sprintf("x/%s/types/genesis_test.go", opts.ModuleName)
+		f, err := r.Disk.Find(path)
+		if err != nil {
+			return err
+		}
+
+		templateTests := `%[1]v
+{
+	desc:     "duplicated %[2]v",
+	genState: &types.GenesisState{
+		%[3]vList: []types.%[3]v{
+			{
+				Id: 0,
+			},
+			{
+				Id: 0,
+			},
+		},
+	},
+	valid:    false,
+},
+{
+	desc:     "invalid %[2]v count",
+	genState: &types.GenesisState{
+		%[3]vList: []types.%[3]v{
+			{
+				Id: 1,
+			},
+		},
+		%[3]vCount: 0,
+	},
+	valid:    false,
+},`
```

Comment:
maybe we should add valid test cases too

Reply:
I made the change so we have a single case with all correct fields